three-dna/dist/three.dna.js

```
@@ -0,0 +1,635 @@
+'use strict';
+
+/*
+ThreeDna 0.0.1
+license: MIT
+https://github.com/MichaelMass/three-science
+*/
+
+var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol ? "symbol" : typeof obj; };
+
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
+
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+
+var ThreeDna = function () {
+  function ThreeDna(opts) {
+    _classCallCheck(this, ThreeDna);
+
+    opts = !!opts ? opts : {};
+
+    this._params = {};
+    this._opts = {};
+    this._view = {};
+    this._obj = {};
+    this._obj.mat = {};
+    this._obj.geo = {};
+
+    this._container = opts.container && ThreeDna.Utils.isElement(opts.container) ? opts.container : ThreeDna.Utils._createContainer();
```

It's as you please but I think that the only thing that should be mandatory to pass as an argument is the container element ... in this way you wouldn't need to implement all these helper functions ... You could throw an Error if the element is not an element instead of creating one for the user ..